### PR TITLE
Rich Text Editor Applied

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Added Rich Text Editor Lib.
-Added ReactQuill & its features to Notes App.

### CODE REVIEW

File: Gravity-Notes/src/Pages/Archive/Archive-page.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ import "./Archive.css";
 import { Footer } from "../../Components/Footer/Footer";
 import { ArchiveCard } from "../../Components/Archive-card/ArchiveCard";
 import { useNotes } from "../../Contexts/NotesAction-context";
+import { MdOutlineArchive } from "react-icons/md";
```
Comment (anupkgurung) [inline Gravity-Notes/src/Pages/Archive/Archive-page.js:8]: MdOutlineArchive can have better name

File: Gravity-Notes/src/Components/TextArea.js/Textarea.css [hunk 2]
```diff
@@ -104,4 +106,22 @@
...
+    height: 8rem;
+    color: black;
+}
+.no-note-txt{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 8rem;
+    margin-right: 13rem;
```
Comment (anupkgurung) [inline Gravity-Notes/src/Components/TextArea.js/Textarea.css:120]: use short hand

**Review by anupkgurung — APPROVED** [review]
left few suggestion.

File: Gravity-Notes/src/Components/Trash-card/TrashCard.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ export const TrashCard = ({trash})=>{
             <h1 className="title-txt">{trash.title}</h1>
             <small>{trash.data}</small>
             </div>
-            <p>{trash.content}</p>
+            <p dangerouslySetInnerHTML={{ __html: trash.content}}/>
```
Comment (bhavyaj12) [inline Gravity-Notes/src/Components/Trash-card/TrashCard.js:28]: Liked this

File: Gravity-Notes/src/Pages/Landing-page/landing.css [hunk 4]
```diff
@@ -92,7 +91,7 @@ header .container{
     align-items: center;
 }
 .links a{
-    color: var(--text-color);
+    color: black;
```
Comment (bhavyaj12) [inline Gravity-Notes/src/Pages/Landing-page/landing.css:94]: Use variables preferably

**Review by bhavyaj12 — APPROVED** [review]
Code is easy to understand.
Well structured files. But no deploy preview, it has failed so recheck that once

**Pull request merged**